main.py

```diff
@@ -1,10 +1,52 @@
-import init_django_orm  # noqa: F401
+import init_django_orm  # noqa
 
-from db.models import Race, Skill, Player, Guild
+import json
+from db.models import Race, Skill, Guild, Player
+
+
+def create_race(data: dict) -> Race:
+    race_name = data["name"]
+    race, created = Race.objects.get_or_create(
```

Comment:
You could skip the `created` variable with `_` since it is not used in the logic.